### PR TITLE
feat: analysis for 2.1

### PR DESCRIPTION
**Pull request by Abemelech**
# Question 2.1
Identify data measures based on ratings and reviews that are most informative for Sunshine Company to track, once their three products are placed on sale in the online marketplace

### CODE REVIEW

File: md_question_one.ipynb [hunk 1]
```diff
@@ -0,0 +1,1005 @@
+{
```
Comment (deepakkrish212) [inline md_question_one.ipynb:1]: <p>question: why do we have review body words as components in both of the graphs? What does this tell us exactly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/3/discussion/#comment-1590431653'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (deepakkrish212) [inline md_question_one.ipynb:1]: <p>praise: this is good for question 3 as you mentioned :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/3/discussion/#comment-1590431654'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='27'/>

**Comment by deepakkrish212** [comment]
@Abemelech I had a couple of question regarding the PR but other than that, it looks good to me. I am still not sure what exact "data measure" we are gonna use to track the product's success in the market once after release

File: md_question_one.ipynb (hunk 1 above)
Comment (Abemelech) [reply md_question_one.ipynb:1]: Discussed. I will explain this in detail in the presentation and the importance of the variance.

**Pull request merged**